.travis.yml

```diff
@@ -78,19 +78,22 @@ script:
           /bin/bash -c "GOARCH=amd64 PASSES='fmt bom dep' ./test"
         ;;
       linux-amd64-integration-1-cpu)
+        # TODO: Reenable 'race' when https://github.com/etcd-io/etcd/issues/12336 fixed.
         docker run --rm \
           --volume=`pwd`:/go/src/go.etcd.io/etcd gcr.io/etcd-development/etcd-test:go${TRAVIS_GO_VERSION} \
-          /bin/bash -c "GOARCH=amd64 CPU=1 PASSES='integration' ./test"
+          /bin/bash -c "GOARCH=amd64 CPU=1 PASSES='integration' RACE='false' ./test"
```

Comment:
How flaky is `TestV3WatchRestoreSnapshotUnsync` currently? My concerns is that there might be more race conditions introduced to the codebase in the future, once we disabled race in CI testing.

Reply:
Thanks! You are right. We had race detection for unit pass but not for integration.